gooddata.gemspec

```diff
@@ -49,7 +49,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'vcr'
 
   s.add_dependency 'activesupport', '~> 4.2.9'
-  s.add_dependency 'aws-sdk', '~> 2.9', '>= 2.9.42'
+  s.add_dependency 'aws-sdk-s3', '~> 1'
```

Comment:
Please use the latest version when introducing dependencies `gem 'aws-sdk-s3', '~> 1.16'`.

Reply:
We should update `Gemfile.lock` by running `bundle install`. This will ensure that a specific version is always used.

Reply:
Can you please squash the two commits into one?

Reply:
@panjan I did the proper version locking in d4cec5d57, but I came to realize that the `Gemfile.lock` is ignored by the `.gitignore`, should I remove it from the ignore list and add to the repo anyway?

Reply:
@henrich-m You're right. I've just realised that this repo is distributed as a Gem, therefore the Gemfile.lock should be ignored and only gemspec should be updated. Sorry for the confusion.